base/Data/Universe/Class.hs

```diff
@@ -14,6 +14,13 @@ import Data.Universe.Helpers
 -- recursively enumerable (and that 'universe' is that enumeration). In
 -- particular, you promise that any finite inhabitant has a finite index in
 -- 'universe', and that no inhabitant appears at two different finite indices.
+--
+-- /Laws:/
+--
+-- @
+-- 'elem' x 'universe'                       -- any inhabitant has a finite index
```

Comment:
aligment of comments, seems of, but it's due different amount of markup quotes. Comments are aligned when rendered.